auto_inertial_params/proposal.md

```diff
@@ -0,0 +1,410 @@
+
+Similarly, the Products of Inertia can be calculated as:
+
+$$\begin{eqnarray}
+ I\_{12}  = I\_{xy} = \int -xydm = I\_{yx} = I\_{21} \\\
+ I\_{13}  = I\_{xz} = \int -xzdm = I\_{zx} = I\_{31} \\\
+ I\_{23}  = I\_{yz} = \int -yzdm = I\_{zy} = I\_{32} \\\
+\end{eqnarray}$$
+
+In this solution, the **infinitesimal element** of the object can be **represented by the each Voxel.** 
+Instead of calculating the mass \\(dm\\) for each voxel, we will calculate the volume \\(dv\\) of each 
+voxel using the voxel size (the simple volume formula of a cube can be used because voxels are cubes). 
+Then mass for each element would be mass density, \\(\rho\\) multiplied by the volume, \\(dv\\). 
+Considering the mass density to be constant and substituting in the above equations we get:
+
+$$\begin{eqnarray}
+I\_{11} = I\_{xx} = \rho\int (y^2 + z^2)dv \\\
+I\_{22} = I\_{yy} = \rho\int (x^2 + z^2)dv \\\
+I\_{33} = I\_{zz} = \rho\int (x^2 + y^2)dv \\\
+I\_{12}  = I\_{xy} = \rho\int -xydv = I\_{yx} = I\_{21} \\\
+I\_{13}  = I\_{xz} = \rho\int -xzdv = I\_{zx} = I\_{31} \\\
+I\_{23}  = I\_{yz} = \rho\int -yzdv = I\_{zy} = I\_{32} \\\
+\end{eqnarray}$$
+
+#### Advantages of the Voxelization Approach
+ * Voxelization is a generalized approach that would work for all kinds 
+ of meshes (convex and non-convex meshes).
+ * It can be made configurable by allowing the user to set an appropriate `voxel_size` 
+ for the process. A smaller `voxel_size` would be comparitively computationally heavy 
+ but would provide closer to real world value for moment of inertia.
+ * Voxelization is more intuitive way of moment of inertia calculations as 
+ compared to other integral methods.
+
+### Integration-based numerical method
+It uses Gauss’s Theorem and Greene’s Theorem of integration to convert volume 
+integrals to surface integrals (Gauss’s Theorem) and then surface integrals to line 
+integrals(Greene’s Theorem)<sup>[\[5\]](#References)</sup>.
+This method works for triangle meshes which are simple water-tight polyhedrons. 
+Currently, the origin of the mesh being used needs to be set at the geometric centre 
+to obtain the correct value.
+Since this method uses the vertex data for calculations, a high vertex count would be 
+required for near-ideal values. For eg, in case of a cylinder, it was observed that with 
+a vertex count of 4096, the inertial values obtained were withtin a 0.005 tolerance 
+of the ideal values.
+
+## References
+
+ * [1]: https://en.wikipedia.org/wiki/Voxel
+
+ * [2]: https://davidstutz.de/efficiently-voxelizing-watertight-meshes-into-occupancy-grids-and-signed-distance-functions/\
+
+ * [3]: https://towardsdatascience.com/how-to-voxelize-meshes-and-point-clouds-in-python-ca94d403f81d
+
+ * [4]: https://blog.spatial.com/the-main-benefits-and-disadvantages-of-voxel-modeling
+
+ * [5]: https://www.geometrictools.com/Documentation/PolyhedralMassProperties.pdf
```

Comment:
List is not being rendered

Reply:
It seems to be rendered correctly for me. 
![Screenshot from 2023-09-09 00-51-54](https://github.com/gazebosim/sdf_tutorials/assets/23265149/79055753-706b-4dd8-b465-b79063b1a1c1)
Are you referring to the bullets and numbers both being rendered? I tried removing the bullets but that resulted in the list not being rendered at all.